deps/rabbitmq_web_stomp/src/rabbit_web_stomp_handler.erl

```diff
@@ -233,21 +204,37 @@ websocket_info({start_heartbeats, {SendTimeout, ReceiveTimeout}},
     ReceiveFun = fun() -> Self ! client_timeout end,
     Heartbeat = rabbit_heartbeat:start(SupPid, Sock, SendTimeout,
                                        SendFun, ReceiveTimeout, ReceiveFun),
-    {ok, State#state{heartbeat = Heartbeat}};
+    {ok, State#state{heartbeat = Heartbeat,
+                     timeout_sec = {0, 0}}};
+websocket_info(connection_created, State) ->
+    Infos = infos(?INFO_ITEMS ++ ?OTHER_METRICS, State),
+
+    ?LOG_INFO("Connection created infos ~p", [Infos]),
```

Comment:
Logging the connection infos at `info` level of every Web STOMP connection pollutes the logs too much.